README.md

```diff
@@ -64,36 +64,9 @@ data:
 
 #### Rebuilding the image
 
-First to clone the repository and update the anki-sync-server:
-
-    git clone https://github.com/ankicommunity/docker-anki-sync-server
-    cd docker-anki-sync-server
-
-Then apply the following patch to `anki-sync-server/Docker/anki-sync-server`:
-
-```patch
-diff --git a/src/requirements.txt b/src/requirements.txt
-index 77ba117346c8..f78905c62e23 100644
---- a/src/requirements.txt
-+++ b/src/requirements.txt
-@@ -1,8 +1,5 @@
--THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!
--
--Warning: The lock file is not up to date with the latest changes in pyproject.toml. You may be getting outdated dependencies. Run update to update them.
--anki==2.1.36; python_version >= "3.7"
--ankirspy==2.1.36; python_version >= "3.7"
-+anki==2.1.35; python_version >= "3.7"
-+ankirspy==2.1.35; python_version >= "3.7"
- beautifulsoup4==4.9.1
- certifi==2020.6.20; python_version >= "3.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "3.7"
- chardet==3.0.4; python_version >= "3.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "3.7"
-```
-
-Enter the Docker directory and alter `build.sh` to suit your use case. Run it to build the image on the current platform.
+From the root of the repository, run `scripts/build_devops.sh` optionally specifying which project you want to build:
 
-    cd Docker
-    vi build.sh
-    ./build.sh
+    SERVICE="anki-sync-server" scripts/build-devops.sh;
```

Comment:
Can we use the make command: 
```
$ make build
``` 

and for a specific service:
```
$ make build SERVICE=anki-sync-server
```
the benefit of this is the script will run with all the environment variables in the  `config/.env.local` folder includes all the build args for configuration. If that file does not exist, then they should create one e.g.
```
$ cp config/.env.example config/.env.local
```

Reply:
Sure thing! `make` seems like a silly way to make sure environment variables are loaded since we're not exactly making anything and shell scripts eat this stuff for breakfast, but if other repos do it this way, I'm happy to oblige. Likewise 
doing `make build SERVICE=...` is pretty surprising to anyone who knows howto use a shell, but for consistency, so be it.

I pushed a new commit. Let me know if anything else can be improved.

Reply:
Yh the `make` commands are just my prefered abstraction for a build system. The reason I use it is because then any new user can use `make help` to see a list of all the available options. Thus reducing the barrier to entry.